docs/Developers/Newcomers_Guide.rst

```diff
@@ -34,6 +34,8 @@ Once you've completed all five tasks, please fill our
 `promotion request form <https://docs.google.com/forms/d/e/1FAIpQLSevrkrJga23GySFUdngrTEo0CdjXcqjHBHqX2z_JF2Z61YNmw/viewform?usp=sf_link>`_
 to get promoted to the role of developer. GitLab contributions also
 count to the promotion of a newcomer to developer.
+In order to work on GitLab issues, you should join
```

Comment:
I think they need to *request* to join.

Reply:
By requesting access to single GitLab repository, or is there some better way ?

Reply:
I think what @jayvdb is saying is that the sentence should be like:
In order to work on GitLab issues, you should be a member of GitLab newcomers group. You can send a *request* to join this group [here](link to GitLab group)

Reply:
I'm asking, because i don't see any central "request access" button at https://gitlab.com/coala/roles/newcomers.
I can see informations about group being public, and can set notifications but to request access i have to select repository(Project). 
There is no request access for the group, like in https://docs.gitlab.com/ee/user/group/#request-access-to-a-group